### DISCUSSION TITLE
Running samples without normal/tumor pairings.

### DISCUSSION THREAD

**Discussion by raagagrawal** [uclahs-cds/metapipeline-DNA#53] [Q&A] [answered]
I saw in the README that each patient needs a normal sample and a tumor pair. I am running the metapipeline on cell lines so I don't have any normal-tumor pairings. Is it possible to run the pipeline w/ only a single state e.g. all normal or all tumor like it is in the call-gsnp pipeline?

**Comment by tnyamaguchi** [comment, marked as answer]
Yes, single sample mode (both normal or tumor only) is available. ([v3.0.0-rc.1](https://github.com/uclahs-cds/metapipeline-DNA/releases/tag/v3.0.0-rc.1) and above)

**Reply by yashpatel6** [reply]
I'll also add, you can have a mix and run in single-sample mode. Ex. 20 normal samples and 30 tumor samples can all go into the same input CSV and be submitted together in single-sample mode; the appropriate pipelines will be run based on sample type for each individual sample.
